### PR TITLE
Districts Name and Province 6 clean data updated

### PR DESCRIPTION
**Pull request by nikeshbalami**
- Renamed the old district name
- Removed the unused rows and space from province 6 data

### CODE REVIEW

File: Federal Data/753 Local Unit Population and HouseHold/Pradesh 6/Dailekh/Bhagawatimai Gaunpalika.csv [hunk 1]
```diff
@@ -1,12 +1,9 @@
-,,, ,
-,,,,
-Ward No.,Household ,Total Population ,Total Female ,Total Female 
-,,,,
+Ward No.,Household,Total Population,Total Female,Total Female
```
Comment (cliftonmcintosh) [inline Federal Data/753 Local Unit Population and HouseHold/Pradesh 6/Dailekh/Bhagawatimai Gaunpalika.csv]: `Female` twice in the headers. Should be Male, then Female, I believe.

**Review by cliftonmcintosh — CHANGES_REQUESTED** [review]
Some of the files still have "Total " and need the space removed.

File: Federal Data/753 Local Unit Population and HouseHold/Pradesh 6/Dailekh/Bhagawatimai Gaunpalika.csv (hunk 1 above)
Comment (nikeshbalami) [reply Federal Data/753 Local Unit Population and HouseHold/Pradesh 6/Dailekh/Bhagawatimai Gaunpalika.csv]: Yes, that should be Male then Female. I missed it while looking.

**Comment by nikeshbalami** [comment]
Have tried removing all the unused space this time.

**Review by cliftonmcintosh — APPROVED** [review]

**Pull request merged**